app/views/partials/view-claim/add-deduction.html

```diff
@@ -1,22 +1,104 @@
-      </td>
-      <td>
-        <input type="submit" formaction="/claim/{{ Claim['ClaimId'] }}/add-deduction" class="button pull-right grey" name="add-deduction" id="add-deduction" value="Add">
-      </td>
-    </tr>
-  </tbody>
-</table>
-{% endif %}
\ No newline at end of file
+    {% set deductionSelect %}
+    <div class="claim-table-deduction-section-container">
+      <div>
+        <span class="govuk-!-margin-top-2 govuk-!-margin-right-4 govuk-!-font-weight-bold">Type</span>
+      </div>
+      <div>
+        {{ govukSelect({
+          id: "deductionType",
+          name: "deductionType",
+          formGroup: {
+            classes: "govuk-!-margin-bottom-0",
+            id: "deductionType"
+          },
+          errorMessage: errors | findError('deductionType'),
+          items: [
+            {
+              value: "Select",
+              text: "Select",
+              selected: true
+            },
+            {
+              value: "hc3",
+              text: displayHelper.getDeductionTypeDisplayName('hc3')
+            },
+            {
+              value: "overpayment",
+              text: displayHelper.getDeductionTypeDisplayName('overpayment')
+            }
+          ]
+        }) }}
+      </div>
+    </div>
+    {% endset %}
+
+    {% set deductionAmount %}
+    <div class="claim-table-deduction-section-container">
+      <div>
+        <span class="govuk-!-margin-top-2 govuk-!-margin-right-4 govuk-!-font-weight-bold">Amount</span>
+      </div>
+      <div>
+        {{ govukInput({
+          label: {
+            text: "Deduction amount",
+            classes: "govuk-visually-hidden"
+          },
+          errorMessage: errors | findError('deductionAmount'),
+          formGroup: {
+            classes: "govuk-!-margin-bottom-0"
+          },
+          classes: " govuk-input--width-4 ",
+          prefix: {
+            text: "£"
+          },
+          value: "",
+          id: "deductionAmount",
+          name: "deductionAmount",
+          inputmode: "decimal"
+        }) }}
+      </div>
+    {% endset %}
+
+    {% set deductionButton %}
+        {{ govukButton({
+          text: "Add",
+          id: "add-deduction",
+          type: "submit",
+          name: "add-deduction",
+          attributes: {
+            formaction: "/claim/" +  Claim['ClaimId'] + "/add-deduction"
+          },
+          classes: "govuk-!-margin-bottom-0 edit-claim-button govuk-button--secondary pull-right"
+        }) }}      
+    {% endset %}
+
+    {{ govukTable({
+      caption: "Deduction",
+      captionClasses: "govuk-table__caption--l",
+      classes: "govuk-!-margin-bottom-8",
+      rows: [
+        [
+          {
+            html: deductionSelect,
+            classes: "claim-table-width-one-third vertical-align-middle"
+          },
+          {
+            html: deductionAmount,
```

Comment:
So, going back to the CSS naming, maybe this class should be `deduction-amount` and just have the values you're setting here?

Reply:
..and the same for others.

Reply:
My thinking was - theres quite a few variants of classes i've set up in the claim table that use the GOV design classes, so set them individually in the html for clarity